apps/io-func-sign-user/src/signature-field.ts

```diff
@@ -31,12 +31,12 @@ export const toSignatureFieldToBeCreatedAttributes =
     page,
   }: AttributesWithCoordsAndSize): SignatureFieldToBeCreatedAttributes => ({
     bottomLeft: {
-      x: coordinates.x,
-      y: pageHeight - coordinates.y + size.h,
+      x: Math.round(coordinates.x),
+      y: Math.round(pageHeight - coordinates.y + size.h),
```

Comment:
```suggestion
      y: Math.round(pageHeight - coordinates.y -  size.h),
```
we are always thinking from the opposite point of view!